docs/design/resource-strategy-fit-scheduling.md

```diff
@@ -0,0 +1,95 @@
+# Enhanced NodeResourceFit Plugin
+
+## Summary
+
+The NodeResourcesFit plug-in of native k8s can only adopt a type of strategy for all resources, such as MostRequestedPriority and LeastRequestedPriority. However, in industrial practice, this design does not apply to some scenarios. For example: In AI scenarios, businesses that apply for GPUs prefer to occupy the entire GPU machine first to prevent GPU fragmentation; businesses that apply for CPU & MEM are prioritized and dispersed to non-GPU machines to prevent excessive consumption of CPU & MEM on GPU machines, resulting in real tasks of applying for GPUs. Pending due to insufficient non-GPU resources
```

Comment:
Now the name has been already renamed to `NodeStrategyFit`, the doc is stil `NodeResourcesFit`, please check all the names that have not been renamed yet to `NodeStrategyFit`